### PR TITLE
4.0.1 branch

### PR DESCRIPTION
**Pull request by arjantijms**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <groupId>org.glassfish.build</groupId>
     <artifactId>glassfishbuild-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>4.0.1-SNAPSHOT</version>
+    <version>4.0.2-SNAPSHOT-SNAPSHOT</version>
```
Comment (pzygielo) [inline pom.xml:33]: - :eyes: https://github.com/eclipse-ee4j/glassfish-spec-version-maven-plugin/pull/144#issuecomment-2031457337

**Review by dmatej — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (dmatej) [reply pom.xml]: I have fixed that now ...

**Comment by pzygielo** [comment]
@arjantijms Perhaps 4.0.1 could be released to central?

**Comment by arjantijms** [comment]
It is done: https://repo1.maven.org/maven2/org/glassfish/build/glassfishbuild-maven-plugin/4.0.1/

**Pull request merged**